.eslintrc.js

```diff
@@ -29,6 +29,7 @@ module.exports = {
   settings: {
     "import/resolver": {
       "node": {
+        "paths": ['src'],
```

Comment:
Can you explain why this branch needs this change?

Reply:
Nvm, I think I get it now. It provides a base resolution path so we can get away from the `..` nonsense. Cool! Good change.

Reply:
yes, that's right.